dht-node/src/dht_node/index.test.ts

```diff
@@ -710,17 +719,17 @@ describe('federation', () => {
                   Buffer.from(
                     JSON.stringify([
                       {
-                        api: 'v1_0',
-                        url: 'http://localhost:4000/api/v1_0',
+                        api: '1_0',
+                        url: 'http://localhost:5001/api/',
```

Comment:
Like written in the other file I prefer the logic with the API version in the URL instead of making some port magic

Reply:
as answered on your comment before, the endpoint, which will be used to communitcate with the other community will follow the pattern \<host\>:\<port\>/api/<version-nr>/
the url in the database of a foreign community defines this part \<host\>:\<port\>/api/
the api in the database of the foreign community will be appended on the url.
so the resulting endpoint which the grphql-client will invoke looks like
\<host\>:\<port\>/api/\<version-nr\>/

the magic on the port is a requirement from Ulf and Moritz...